package.json

```diff
@@ -42,7 +42,7 @@
     "test": "npm run test:unit && npm run test:e2e:smoke",
     "test:unit": "npm run test:main && npm run test:preload && npm run test:preload-docker-extension && npm run test:renderer && npm run test:tools && npm run test:extensions",
     "test:e2e:smoke": "cross-env NODE_ENV=development MODE=development DEBUG=pw:browser npm run build && xvfb-maybe vitest run tests/src/e2e-smoke.spec.ts",
-    "test:main": "vitest run -r packages/main --passWithNoTests --coverage",
+    "test:main": "vitest run -r packages/main --passWithNoTests --coverage --no-threads",
```

Comment:
why this ?
The PR is not speaking about this change

Reply:
yes but why should we do that ?
If each time a dependency is changed we change the number of threads of vitest ?
It has impacts on the speed of the tests

It is also changing default behaviour: IDE may still use another value ( if you click on run tests of a file) etc

Reply:
https://github.com/vitest-dev/vitest/issues/3143#issuecomment-1621072334

It seems you can only change something for impacted tests instead of changing for all the tests

Reply:
There is no way to identify the test at fault, because it looks like it can be any. Impact is around 25%, but we have not so many tests for main package, so in my env it is about 0.5 sec. 

I assumed that new deps bring this error and checked if I can downgrade the version of some of them and get it back to work. I found the reason for segmentation fault is ssh2. Resolving it to v1.11.0, which is lowest docker-modem is requiring, fixes the problem.

PR is updated and tested on ubuntu 23.